dev-requirements.txt

```diff
@@ -1,3 +1,4 @@
+itsdangerous==0.24
```

Comment:
This is only for dependencies used during development (testing, etc). It's not for consumers of the lib.